src/commands/autocomplete/create.ts

```diff
@@ -184,8 +184,89 @@ compinit;\n`
 
   private get bashCompletionFunction(): string {
     const cliBin = this.cliBin
+    let bashScript = ''
 
-    return `#!/usr/bin/env bash
+    if (this.config.topicSeparator === ' ') {
+      bashScript = `#!/usr/bin/env bash
```

Comment:
Do you mind moving these strings somewhere else so we can keep this more readable, e.g. 

```typescript
const bashScript = this.config.topicSeparator === ' ' ? bashScriptForSpaces : bashScriptForColons
```